frontend/src/api/mockAPI/utils.js

```diff
@@ -245,6 +246,18 @@ export class MockAPIController {
     validateNew() {
         throw new Error("Subclasses must impliment `validateNew()`");
     }
+
+    /**
+     * Validates a property of an item instance. Throws an error
+     * if this property are invalid/incompatable.
+     *
+     * @param {object} obj
+     * @param {string} prop
+     * @memberof MockAPIController
+     */
+    validateProp(obj, prop) {
+        if (obj[prop]) return true;
```

Comment:
Never, ever, ever use an if statement without `{...}` around it's following block.

Reply:
I see, if I don't use obj or prop anywhere inside this function, prettier would complain(fail the formatting tests). but both obj and prop are checked before even call this function. what do you think I should do here

Reply:
You can remove `obj` and `prop` from the function signature. Or you can add a `// eslint-ignore-next-line` (or whatever it is..) comment.